Binary files orig/StanHeaders/R/.Flags.R.swp and patched/StanHeaders/R/.Flags.R.swp differ
diff -Nru orig/StanHeaders/R/Flags.R patched/StanHeaders/R/Flags.R
--- orig/StanHeaders/R/Flags.R	2024-03-04 23:33:33.000000000 +0100
+++ patched/StanHeaders/R/Flags.R	2024-08-13 10:05:11.122850100 +0200
@@ -23,7 +23,10 @@
     TBB_LIB <- system.file("lib", .Platform$r_arch, package = "RcppParallel", mustWork = TRUE)
   }
 
-  PKG_LIBS <- paste0("-L", shQuote(TBB_LIB), " -Wl,-rpath,", shQuote(TBB_LIB), " -ltbb -ltbbmalloc")
+  PKG_LIBS <- paste0("-L", shQuote(TBB_LIB), 
+		     if (.Platform$OS.type != "windows")
+			     paste0(" -Wl,-rpath,", shQuote(TBB_LIB)),
+		     " -ltbb -ltbbmalloc")
 
   if (isTRUE(as_character)) return(PKG_LIBS)
   cat(PKG_LIBS, " ")
